mlflow/store/file_store.py

```diff
@@ -360,10 +360,6 @@ def create_run(self, experiment_id, user_id, run_name, source_type,
         mkdir(run_dir, FileStore.ARTIFACTS_FOLDER_NAME)
         for tag in tags:
             self.set_tag(run_uuid, tag)
-        if parent_run_id:
-            self.set_tag(run_uuid, RunTag(key=MLFLOW_PARENT_RUN_ID, value=parent_run_id))
-        if run_name:
-            self.set_tag(run_uuid, RunTag(key=MLFLOW_RUN_NAME, value=run_name))
         return Run(run_info=run_info, run_data=None)
 
     def get_run(self, run_id):
```

Comment:
so we're not supporting reading of the deprecated fields for runs saved in the old format (e.g. by returning them in tags) right?

Reply:
This is a good point -- potentially we should consider having a migration script to change the old runs to include tags.